Orange/widgets/evaluate/owconfusionmatrix.py

```diff
@@ -426,11 +426,13 @@ def sum_item(value, border=""):
     def send_report(self):
         if self.results is not None and self.selected_learner:
             index = self.selected_learner[0]
+            cmatrix = confusion_matrix(self.results, index)
+            n= len(cmatrix)
```

Comment:
It's likely the number of distinct class values. To avoid computing confusion matrix all over, could the value be devised from `self.tablemodel`?